### PR TITLE
Buy Now NFTs and NFT Splits

### PR DESCRIPTION
**Pull request by lazynina**
Mint NFT modal
image

Additional Royalties in NFT Modal
image

Put NFT on sale modal
image

Buy Now Price displayed if only 1 copy
image

Place Bid Modal
image

Additional Royalties specified in Place Bid Modal
image

### CODE REVIEW

File: src/app/backend-api.service.ts [hunk 1]
```diff
@@ -340,6 +344,8 @@ export class NFTEntryResponse {
   IsPending?: boolean;
   MinBidAmountNanos: number;
   LastAcceptedBidAmountNanos: number;
+  IsBuyNow: number;
```
Comment (superzordon) [inline src/app/backend-api.service.ts:347]: Should this be a boolean?

Comment (lazynina) [reply src/app/backend-api.service.ts:347]: Yes!

Comment (lazynina) [inline src/app/backend-api.service.ts:347]:
```suggestion
  IsBuyNow: boolean;
```

**Review by diamondhands0 — APPROVED** [review]

File: src/app/notifications-page/notifications-list/notifications-list.component.ts [hunk 2]
```diff
@@ -277,28 +295,66 @@ export class NotificationsListComponent {
...
-          }`
-        : `${actorName} cancelled their bid on serial number ${nftBidMeta.SerialNumber}`;
-      result.icon = nftBidMeta.BidAmountNanos ? "fas fa-dollar-sign fc-blue" : "fas fa-dollar-sign fc-red";
       result.bidInfo = { SerialNumber: nftBidMeta.SerialNumber, BidAmountNanos: nftBidMeta.BidAmountNanos };
-      return result;
-    } else if (txnMeta.TxnType == "ACCEPT_NFT_BID") {
+      result.post = this.postMap[postHash];
+      if (
+        nftBidMeta.IsBuyNowBid &&
+        this.globalVars.loggedInUser?.PublicKeyBase58Check === nftBidMeta.OwnerPublicKeyBase58check
```
Comment (superzordon) [inline src/app/notifications-page/notifications-list/notifications-list.component.ts:310]: I believe this should be !==

**Pull request merged**